test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_default_config_test.clj

```diff
@@ -80,36 +80,39 @@ react accordingly."
                           true
                           false)
           client        (.createHttpClient proxy-servlet)]
-      ;; See: https://github.com/eclipse/jetty.project/blob/jetty-9.2.10.v20150310/jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java#L129
+      ;; See: https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java#L135
       (is (= 4096 (.getRequestBufferSize client))
           "Unexpected default for proxy 'request-buffer-size'")
-      ;; See: https://github.com/eclipse/jetty.project/blob/jetty-9.2.10.v20150310/jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java#L268-L271
+      ;; See: https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java#L304-L307
       (is (= 30000 (.getIdleTimeout client))
           "Unexpected default for proxy 'idle-timeout'")
       (.stop client))))
 
 (def selector-thread-count
   "The number of selector threads that should be allocated per connector.  See:
-   https://github.com/eclipse/jetty.project/blob/jetty-9.2.10.v20150310/jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java#L229"
-  (max 1 (min 4 (int (/ (ks/num-cpus) 2)))))
+   https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java#L223
+   and
+   https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java#L403-L408
+   The number of selectors is twice the number of selector threads."
+  (* 2 (max 1 (min 4 (int (/ (ks/num-cpus) 2))))))
```

Comment:
Would be good to update this in the description of the [ssl-selector-threads](https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/1.7.0/doc/jetty-config.md#ssl-selector-threads) webserver option too.